client/app/bundles/course/assessment/submission/components/ScribingView/__test__/index.test.tsx

```diff
@@ -65,6 +65,8 @@ describe('ScribingView', () => {
 
     const page = render(<ScribingView answerId={answerId} />, { at: [url] });
 
-    expect(await page.findByTestId(`canvas-${answerId}`)).toBeVisible();
+    expect(
```

Comment:
this commit looks like strongly related to the previous commit (which is to have the `timeout` set up). Better to squash this commit with previous commit

Reply:
This one runs on jstest, not playwright, hence separated.